src/main/webapp/admin_dashboard_script.js

```diff
@@ -29,166 +29,176 @@ window.addEventListener("load", getUserTasks());
+				stylers: [{ color: "#6b9a76" }],
+			},
+			{
+				featureType: "road",
+				elementType: "geometry",
+				stylers: [{ color: "#38414e" }],
+			},
+			{
+				featureType: "road",
+				elementType: "geometry.stroke",
+				stylers: [{ color: "#212a37" }],
+			},
+			{
+				featureType: "road",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#9ca5b3" }],
+			},
+			{
+				featureType: "road.highway",
+				elementType: "geometry",
+				stylers: [{ color: "#746855" }],
+			},
+			{
+				featureType: "road.highway",
+				elementType: "geometry.stroke",
+				stylers: [{ color: "#1f2835" }],
+			},
+			{
+				featureType: "road.highway",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#f3d19c" }],
+			},
+			{
+				featureType: "transit",
+				elementType: "geometry",
+				stylers: [{ color: "#2f3948" }],
+			},
+			{
+				featureType: "transit.station",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#d59563" }],
+			},
+			{
+				featureType: "water",
+				elementType: "geometry",
+				stylers: [{ color: "#17263c" }],
+			},
+			{
+				featureType: "water",
+				elementType: "labels.text.fill",
+				stylers: [{ color: "#515c6d" }],
+			},
+			{
+				featureType: "water",
+				elementType: "labels.text.stroke",
+				stylers: [{ color: "#17263c" }],
+			},
+		],
+	});
+	//Resize Function
+	google.maps.event.addDomListener(window, "resize", function () {
+		var center = map.getCenter();
+		google.maps.event.trigger(map, "resize");
+		map.setCenter(center);
+	});
 }
 
 function load(file) {
-  var src = document.createElement("script");
-  src.setAttribute("type", "text/javascript");
-  src.setAttribute("src", file);
-  document.getElementsByTagName("head")[0].appendChild(src);
+	var src = document.createElement("script");
+	src.setAttribute("type", "text/javascript");
+	src.setAttribute("src", file);
+	document.getElementsByTagName("head")[0].appendChild(src);
 }
 
 function getUserTasks() {
-  fetch("/user-tasks")
-    .then((response) => response.json())
-    .then((tasks) => {
-      
-      console.log(tasks);
-      userTasksArray = tasks;
-      let taskSection = document.getElementById("tasks-container");
-      for (userTask of tasks) {
-        taskSection.innerHTML += addTask(userTask);
-      }
-    });
+	fetch("/user-tasks")
+		.then((response) => response.json())
+		.then((tasks) => {
+			console.log(tasks);
+			userTasksArray = tasks;
+			let taskSection = document.getElementById("user-tasks");
+			for (userTask of tasks) {
+				taskSection.innerHTML += addTask(userTask);
+			}
+		});
 }
 
 function addTask(task) {
-  let string = `<a href="#popup-overlay"><li class="admin-task user-task"  onclick="openWithPopup(${task.key.id})"><h3> ${task.propertyMap.category} </h3>`;
-  string += `<h4> ${task.propertyMap.Owner} </h4></li></a>`;
-  return string;
+	let string = `<a href="#popup-overlay"><li class="user-task"  onclick="openWithPopup('${task.keyString}')"><h3> ${task.category} </h3>`;
```

Comment:
Same suggestion here: Maybe "admin-task" is a better name than "user-task" for the class

Reply:
renamed to admin-user-task